README.md

```diff
@@ -1 +1,64 @@
-# first-capstone-summit
\ No newline at end of file
+# html-css-js-capstone-project
+
+This is my final project for HTML & CSS MODULE from the microverse curriculum
+
+<p align="center">
+  <img src="./img/screenshotmobile.png"/>
+</p>
+
+<hr>
+
+<p align="center">
+  <img src="./img/screenshotdesktop.png"/>
+</p>
+
+This project is meant to test student ability's on being able to put together all that they have learnt in a module to ensure they mastered it well.
+
+## Built With
+
+- HTML
+- CSS (SASS)
+- JAVASCRIPT
+
+## Live Demo
+
+[Live Demo Link](https://ndayishimiyeeric.github.io/first-capstone-summit/)
```

Comment:
- [ ] The live demo link is broken. Please use GitHub Pages to deploy your project and add the link to the deployed project to your README.